Troubleshooting/resolving-peer-rejected.md

```diff
@@ -1,5 +1,11 @@
-Peer Rejected is a state a peer can be in. If you see a peer in this
-state when you run 'gluster peer status' that means volume configuration
+##Fixing "Peer Rejected" 
```

Comment:
A general note: If there isn't a space between `#` and start of the heading text, [GitHub doesn't render it as a heading](https://github.com/github/markup/issues/1013). You will see this broken across all documentation currently. As this is a new code, please add the space.

We should think of a way to automate bulk fixing this in all files.

Reply:
Fixed for this instance.